spec/features/browsing_item_spec.rb

```diff
@@ -51,4 +51,24 @@
       end
     end
   end
+
+  context 'when an item does not have a format value' do
+    before do
+      visit 'catalog/SCSB-7935196'
+    end
+
+    it 'has a default icon even if there is no fomat value' do
```

Comment:
If the test just looked for that selector it would have passed without this PR and while not displaying the icon. The only way to know the icon has been applied is the content of the ::before pseudo element.

What do you dislike about using a snippet of JS in the test? We use page.evaluate_script already in another test, albeit without the full script block.